CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.49.1
+
+## Fixed
+
+* Fixed: `Invoke-WhiskeyTask` will install required tools before changing task's working directory

```

Comment:
Change to `Fixed: Whiskey fails to install a task's tool if the task is running with a custom working directory.`